test/site/_collections/index.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
+const path = require('path');
+const runEleventy = require('./runEleventy');
+
+describe('_collections', function() {
+  describe('_collections in DEV env', function() {
+    before(async function() {
+      await runEleventy('dev', '.');
+      console.log('Eleventy build finished. Starting tests...');
+    });
+
+    after(function() {
+      fs.rmdirSync(path.join('.', '.tmp'), {recursive: true});
+    });
+
+    require('./posts-with-lighthouse/dev.spec');
+    require('./recent-posts/dev.spec');
+  });
+
+  describe('_collections in PROD env', function() {
+    before(async function() {
+      await runEleventy('prod', '.');
+      console.log('Eleventy build finished. Starting tests...');
+    });
+
+    after(function() {
+      fs.rmdirSync(path.join('.', '.tmp'), {recursive: true});
+    });
+
+    require('./posts-with-lighthouse/prod.spec');
+    require('./recent-posts/prod.spec');
+  });
+});
```

Comment:
Note there's also a `PERCY` env that changes the behavior of `livePosts`. We should test that as well.

Reply:
Noted. Will try to test in in livePosts test then.